modules/ROOT/pages/index.adoc

```diff
@@ -8,26 +15,151 @@
+
+[.link]
+xref:installation/index.adoc[]
+
+[.widget.lists]
+== Tailored for you
+
+[.caption]
+Keep exploring
+
+=== Neo4j admin commands
+
+[.icon]
+image:icon-admin.svg[]
+
+[.list]
+* xref:/tools/neo4j-admin/consistency-checker.adoc[`database check`]
+* xref:/tools/neo4j-admin/neo4j-admin-report.adoc[`server report`]
+* xref:/tools/neo4j-admin/neo4j-admin-store-info.adoc[`database info`]
+* xref:/tools/neo4j-admin/neo4j-admin-memrec/[`server memory-recommendation`]
+* xref:/tools/neo4j-admin/neo4j-admin-import/[`database import`]
+* xref:/tools/neo4j-admin/unbind.adoc[`server unbind`]
+* xref:/tools/neo4j-admin/upload-to-aura.adoc[`database upload`]
+* xref:/tools/neo4j-admin/migrate-database.adoc[`database migrate`]
+* xref:/tools/neo4j-admin/migrate-configuration.adoc[`nserver migrate-configuration`]
+* xref:/tools/neo4j-admin/validate-config.adoc[`server validate-config`]
+* xref:/backup-restore/online-backup.adoc[`database backup`]
+* xref:/backup-restore/aggregate.adoc[`database aggregate`]
+* xref:backup-restore/inspect.adoc[`backup inspect`]
+* xref:/backup-restore/restore-backup.adoc[`database restore`]
+* xref:/backup-restore/offline-backup.adoc[`database dump`]
+* xref:/backup-restore/restore-dump.adoc[`database load`]
+* xref:/backup-restore/copy-database.adoc[`database copy`]
+
+=== Cypher admin commands
+
+[.icon]
+image:icon-developer.svg[]
+
+[.list]
+* xref:database-administration/standard-databases/naming-databases.adoc[Manage standard databases]
+* xref:database-administration/aliases/naming-aliases.adoc[Manage database aliases]
+* xref:database-administration/composite-databases/concepts.adoc[Manage composite databases]
+* xref:clustering/servers.adoc[Manage servers in a cluster]
+* xref:clustering/databases.adoc[Manage databases in a cluster]
+* xref:authentication-authorization/manage-users.adoc[Manage users]
+* xref:authentication-authorization/manage-roles.adoc[Manage roles]
+* xref:authentication-authorization/manage-privileges.adoc[Manage privileges]
+* xref:authentication-authorization/built-in-roles.adoc[Built-in roles and privileges]
+* xref:authentication-authorization/immutable-roles-privileges.adoc[Manage immutable roles and privileges]
+
+=== Reference
+
+[.icon]
+image:icon-cheatsheet.svg[]
+
+[.list]
+* xref:/configuration/configuration-settings.adoc[Configuration settings]
+* xref:procedures.adoc[Procedures]
+* xref:/monitoring/metrics/index.adoc[Metrics]
+* xref:/database-administration/syntax.adoc[Database management commands syntax]
+* xref:/clustering/server-syntax.adoc[Server management commands syntax]
+* xref:/docker/ref-settings.adoc[Docker configuration settings conversion table]
+* xref:/configuration/ports.adoc[Ports in Neo4j]
+* xref:/tools/neo4j-admin/index.adoc[Neo4j Admin and Neo4j CLI]
+* xref:changes-deprecations-removals.adoc[Changes, deprecations, and removals]
+
+=== Cloud object storage integration (COSI)
+
+[.icon]
+image:icon-cloudstorage.svg[]
+
+[.list]
+* Restore from the cloud
```

Comment:
Links needed

Reply:
I'll go through the list and provide links. But there was a question from @renetapopova if we need this column at all.